### ISSUE TITLE
No Backend

### ISSUE THREAD

**Issue by Megamemnon** [Omega-Numworks/Omega#297]
#### Describe the problem
I have no problem following the instructions to install the arm toolchain, clone the Git repository, and build epsilon.elf. But I cannot install. When I run 'make epsilon_flash' I get a "No Backend" error. I have USB drivers - I can successfully flash my Numworks calc from Numworks (with Epsilon) as well as from Omega's install page.

I have tried this on both WSL - following all the Linux instructions - as well as Msys2 on Windows.

Any help would be appreciated, including instructions to simply flash a compiled epsilon.elf file. I tried the Omega Web Installer but get the error that the device is not connected, even though I can seemingly connect to it just fine using the "file:// wants to connect" dialog box (same dialog box as the Numworks and Omega install pages, which work). The top right corner of this page reads "NotFoundError: the device was disconnected". 

I'll play more with the USB drivers; that's sure to keep me busy.

Thanks for any help.

#### Logs
```
Traceback (most recent call last):
  File "build/device/dfu.py", line 598, in <module>
    main()
  File "build/device/dfu.py", line 575, in main
    list_dfu_devices(__Identifiers)
  File "build/device/dfu.py", line 472, in list_dfu_devices
    devices = get_dfu_devices_with_vid_pids(__Identifiers)
  File "build/device/dfu.py", line 100, in get_dfu_devices_with_vid_pids
    devices = [get_dfu_devices(idVendor=vid_pid["VID"],idProduct=vid_pid["PID"]) for vid_pid in vid_pids]
  File "build/device/dfu.py", line 100, in <listcomp>
    devices = [get_dfu_devices(idVendor=vid_pid["VID"],idProduct=vid_pid["PID"]) for vid_pid in vid_pids]
  File "build/device/dfu.py", line 432, in get_dfu_devices
    custom_match=FilterDFU(), **kwargs))
  File "C:\msys64\home\brian\Omega\build\device\usb\core.py", line 1265, in find
    raise NoBackendError('No backend available')
usb.core.NoBackendError: No backend available

```

#### Environment
 - Omega Version: 1.19 0-0
 - OS: Windows 10 (both WSL using the Linux instructions, and Msys2)

**Comment by Megamemnon** [comment]
Figured it out.

I didn't have dfu-util installed. The actual dfu utility is never mentioned in any of the documentation I read. But eventually, I found the following page that mentioned it:

https://en.m.wikibooks.org/wiki/Unofficial_Guide_To_Expanding_Your_Numworks/Getting_your_development_environment_set_up

I used Msys2, this time, and installed the dif-util executable in /usr/local/bin.

I intend to also try installing dfu-util in WSL since I prefer WSL and only installed Msys2 because I couldn't get my Numworks calculator flashed. 

Hopefully this is valuable information to someone else.
